ui/src/app/pages/datasets/add/dataset.add.component.ts

```diff
@@ -187,11 +218,11 @@ export class DatasetAddComponent {
   getAvailableAgentPolicies() {
     return new Promise((resolve) => {
       this.loading[CONFIG.POLICIES] = true;
-      const pageInfo = { ...AgentPoliciesService.getDefaultPagination(), limit: 100 };
```

Comment:
this component was only getting the first 100 policies and sinks for user to choose from when creating/editing dataset - fixed